nautobot/core/management/commands/post_upgrade.py

```diff
@@ -99,6 +99,7 @@ def handle(self, *args, **options):
             self.stdout.write("Performing database migrations...")
             call_command(
                 "migrate",
+                skip_checks=False,  # make sure Postgres version check and others are applied
```

Comment:
Is this a noop? It looks like this is already the default for `migrate.py`

Reply:
No, it's needed. When you use `call_command` it sets `skip_checks=True` if it wasn't already explicitly set by the caller - I'm guessing this is to avoid, in the more common case, re-calling checks that were already called by the parent command? However, by default most management commands (such as our `post_upgrade`) do *not* call `database` checks, but `migrate` does, so I do want to tell it to run its checks.

https://github.com/django/django/blob/stable/3.2.x/django/core/management/__init__.py#L178-L179

Reply:
I see. Thanks for the clarification!